src/analysis/calculus/inverse.lean

```diff
@@ -529,57 +529,59 @@ is_open_map_iff_nhds_le.2 $ λ x, ((hf x).map_nhds_eq (h0 x)).ge
 -/
 
 namespace times_cont_diff_at
-variables {E' : Type*} [normed_group E'] [normed_space ℝ E']
-variables {F' : Type*} [normed_group F'] [normed_space ℝ F']
-variables [complete_space E'] (f : E' → F') {f' : E' ≃L[ℝ] F'} {a : E'}
+variables {𝕂 : Type*} [is_R_or_C 𝕂]
```

Comment:
I'm not sure if it is necessary to be consistent, but fields with `is_R_or_C` seem to have the notation `K` in `data.complex.is_R_or_C` and `𝕜` in (eg) `analysis.normed_space.inner_product`; I don't think `𝕂` has been used before.

Reply:
I don't like `K` when doing analysis (for me, it is a compact set), and `𝕜` was already globally taken, that's why I used `𝕂`.